stable/redis-ha/templates/redis-ha-configmap.yaml

```
@@ -135,3 +135,73 @@ data:
     fi
 
     echo "Ready..."
+{{ if .Values.haproxy.enabled }}
+  haproxy.cfg: |-
+{{- if .Values.haproxy.customConfig }}
+{{ .Values.haproxy.customConfig | indent 4}}
+{{- else }}
+    defaults REDIS
+      mode tcp
+      timeout connect  4s
+      timeout server  30s
+      timeout client  30s
+
+    {{- $root := . }}
+    {{- $fullName := include "redis-ha.fullname" . }}
+    {{- $replicas := int .Values.replicas }}
+    {{- range $i := until $replicas }}
+    # Check Sentinel and whether they are nominated master
+    backend check_if_redis_is_master_{{ $i }}
+      mode tcp
+      option tcp-check
+      tcp-check connect
+      {{- if $root.auth }}
+      tcp-check send AUTH\ {{ .Values.redisPassword }}\r\n
```

Since it is inside `range` we should use `$root` instead of `.Values`.
```suggestion
      tcp-check send AUTH\ {{ $root.redisPassword }}\r\n
```

---

As anal as this is, you're right. I'll commit in a second, thank you!